lib/ae_mdw/db/name.ex

```diff
@@ -162,16 +163,34 @@ defmodule AeMdw.Db.Name do
       map_some(locate_bid(plain_name), &{&1, Model.AuctionBid})
   end
 
-  def pointers(m_name) do
-    case Model.name(m_name, :updates) do
-      [{_, txi} | _] ->
-        Format.to_raw_map(read_tx!(txi)).tx.pointers
-        |> Stream.map(&pointer_kv_raw/1)
-        |> Enum.into(%{})
+  def pointers(Model.name(updates: [])), do: %{}
+
+  def pointers(Model.name(index: plain_name, updates: [{_block_index, txi} | _rest_updates])) do
+    Model.tx(id: tx_hash) = read_tx!(txi)
+    name_hash = :aens.get_name_hash(plain_name)
```

Comment:
not matching because it's a tagged tupple:
```
:aens.get_name_hash("mydev0123456789.chain")
{:ok,
 <<163, 149, 179, 123, 51, 170, 255, 49, 137, 190, 111, 113, 145, 46, 86, 170,
   66, 9, 254, 15, 215, 200, 198, 126, 186, 46, 27, 50, 41, 102, 47, 31>>}
```

Reply:
Done, thanks